graspologic/embed/omni.py

```diff
@@ -145,6 +168,10 @@ class OmnibusEmbed(BaseEmbedMulti):
         Only applicable for ``algorithm="randomized"``; allows you to seed the
         randomized svd solver for deterministic, albeit pseudo-randomized behavior.
 
+    lse : bool, optional (default False)
```

Comment:
I pulled this from the earlier LSE Omni PR -- Is this the api we can all agree on or does someone have other suggestions?

In topologic, I believe we used an enum here (ase or lse, default ase.) Other devs may prefer a magic string but I'd prefer not. 

We can also leave this as is and change it in the future if it turns out we want to support something other than ase/lse.